window.go

```diff
@@ -26,6 +30,26 @@ func (w *Window) AddEventListener(typ string, h EventHandler) {
 	}))
 }
 
+func (w *Window) Open(url, windowName, windowFeatures map[string]string) {
+	w.v.Call("open", url, windowName, joinKeyValuePairs(windowFeatures))
+}
+
+func (w *Window) SetLocation(url string) {
+	w.v.Set("location", url)
+}
+
 func (w *Window) OnResize(fnc func(e Event)) {
 	w.AddEventListener("resize", fnc)
 }
+
+func joinKeyValuePairs(kvpair map[string]string, joiner string) (ret string) {
+	if kvpair == nil {
+		return ""
+	}
+
+	for k, v := range kvpair {
+		ret += k + "=" + v + joiner
```

Comment:
It's better to do this via `strings.Join`

Reply:
will do

Reply:
`strings.Join` is for string list right?